software/messaging/src/main/java/brooklyn/entity/messaging/rabbit/RabbitBrokerImpl.java

```diff
@@ -100,6 +102,14 @@ public void disconnectSensors() {
         disconnectServiceUpIsRunning();
     }
 
+    public boolean getEnableManagementPlugin() {
+        return getConfig(ENABLE_MANAGEMENT_PLUGIN);
```

Comment:
Worth returning `Boolean.TRUE.equals(getConfig(ENABLE_MANAGEMENT_PLUGIN))` just in case someone does something stupid and sets `ENABLE_MANAGEMENT_PLUGIN` to null.